docs/scos/dev/migration-program/migration-to-paas/paas-assessment-documents/how-to-do-the-transition.md

```diff
@@ -0,0 +1,46 @@
+---
+title: How to do the transition?
+description: This document describes how to do the transition.
+template: howto-guide-template
+---
+
+# How to do the transition?
+
+{% info_block infoBox %}
+
+Resources: Backend, DevOps
+
+{% endinfo_block %}
+
+## Description
+
+In this step, we need to assess efforts related to the exact moment of switching the shops from on-prem to Spryker Cloud.
+
+Before you start the transition please make sure that:
+
+1. Depending on a environment type a correspondent template was applied in your cloud service provider (currently, is AWS).
+    All stage similar ones (dev/uat/test/qa etc) should have only one run task per basic service (Yves/Zed/Glue...).
+    For production system this number is 2. Go to ECS -> Clusters -> <your_name> -> tab Services.
+    Check the number in Desired/Running tasks columns.
+2. If you have a production environment, then "Multi AZ" is enabled for RDS/OpenSearch(ES)/Redis(EC) services.
+3. Data is present in RDS based on a dump provided by customer.
+4. All jobs in jenkins are up and running and have green color.
```

Comment:
Aren't the `green color` and `up and running` the same?

Reply:
It's not the same. Jenkins job color could be different depending on a status: success, failed, canceled. pending, in progress, etc.